lib/config/options/index.ts

```diff
@@ -343,6 +343,13 @@ const options: RenovateOptions[] = [
     globalOnly: true,
     default: 'renovate_',
   },
+  {
+    name: 'dockerCliOptions',
+    description:
+      'Docker run command options for use when `binarySource=docker`.',
```

Comment:
```suggestion
      'Pass CLI flags to `docker run` command when `binarySource=docker`.',
```

I think we should put `docker run` in a codeblock style, so it's clear it's about the `docker run` command.